Integrations/integration-fireeye.yml

```diff
@@ -0,0 +1,274 @@
+commonfields:
+  id: fireeye_byoi
+  version: -1
+name: fireeye_byoi
+releaseNotes: "FireEye modified to JavaScript"
```

Comment:
I dont think we need release notes here.
But either: do `releaseNotes: "-"` then we will not add this at all
Or remove this tag, then we will take the description.
In the description you can write that now it supports the new versions of fireeye